mentor_classifier_api/tests/test_fetch.py

```diff
@@ -33,10 +33,11 @@ def test_fetch_data(
 ):
     with open(fixture_path("graphql/{}.json".format(input_mentor))) as f:
         data = json.load(f)
-        responses.add(responses.POST, "http://graphql/graphql", json=data, status=200)
+        responses.add(responses.GET, "http://graphql/graphql", json=data, status=200)
```

Comment:
this needs to be POST. Graphql requests are always POST. That may be masking a problem in local tests, but in real/integrated env, sending a GET requests to graphql would return probably 405 error